lib/util.ex

```diff
@@ -67,27 +73,33 @@ defmodule Util do
       Authorization: "Bearer " <> auth
     ]
 
-    {:ok, resp} = HTTPoison.post("https://api.start.gg/gql/alpha", JSON.encode!(body), headers)
-
-    resp.body
-    |> JSON.decode!()
-    |> Map.get("data")
-    |> Map.get("tournaments")
-    |> Map.get("nodes")
-    # Finesse data into our own non-start.gg format.
-    |> Enum.map(fn sgg_data ->
-      %{
-        "external_id" => "smashgg-" <> Integer.to_string(sgg_data["id"]),
-        "name" => sgg_data["name"],
-        "location" => %{
-          "lat" => sgg_data["lat"],
-          "lng" => sgg_data["lng"]
-        },
-        "start_time" => sgg_data["startAt"],
-        "end_time" => sgg_data["endAt"],
-        "updated_at" => sgg_data["updatedAt"],
-        "url" => "https://start.gg/" <> (sgg_data["slug"])
-      }
-    end)
+    case HTTPoison.post("https://api.start.gg/gql/alpha", JSON.encode!(body), headers) do
+      {:ok, resp} ->
+        case JSON.decode(resp.body) do
+          {:ok, response = %{"success" => false}} ->
+            IO.puts(Map.get(response, "message"))
+          {:ok, response} ->
+            response
+            |> Map.get("data")
+            |> Map.get("tournaments")
+            |> Map.get("nodes")
+            # Finesse data into our own non-start.gg format.
+            |> Enum.map(fn sgg_data ->
+              %{
+                "external_id" => "smashgg-" <> Integer.to_string(sgg_data["id"]),
+                "name" => sgg_data["name"],
+                "location" => %{
+                  "lat" => sgg_data["lat"],
+                  "lng" => sgg_data["lng"]
+                },
+                "start_time" => sgg_data["startAt"],
+                "end_time" => sgg_data["endAt"],
+                "updated_at" => sgg_data["updatedAt"],
+                "url" => "https://start.gg/" <> (sgg_data["slug"])
+              }
+              end )
+        end
+      {:error, reason} -> IO.puts(reason)
```

Comment:
Side note: Any time we print an error, let's print it to stderr with the optional `device` param: https://hexdocs.pm/elixir/1.12/IO.html#puts/2